broker/models/Patient.js

```diff
@@ -34,6 +34,14 @@ Patient {
 }
 */
 
+exports.suffixEnums = {
```

Comment:
HL7 has the suffix as a string, so I don't think we need to validate it other than that. This leaves out stuff like "PhD".

Reply:
Yes, HL7 accepts any value for suffix, but the registration app only allows these specific values.